backstop/tests/header.html

```diff
@@ -620,45 +620,45 @@ <h2>
       <section id="demo-split">
 
         <header class="iui-page-header iui-slim">
-          <div class="iui-left">
+          <div class="iui-page-header-left">
             <!-- Application Button -->
             <div
-              class="iui-header-logo"
+              class="iui-header-brand"
               role="button"
               aria-label="App name"
               aria-expanded="false"
               tabindex="0"
             >
               <svg-imodel-hollow
-                class="iui-header-button-icon"
+                class="iui-header-brand-icon"
                 aria-hidden="true"
               ></svg-imodel-hollow>
-              <span class="iui-label">App name</span>
+              <span class="iui-header-brand-label">App name</span>
             </div>
 
-            <div class="iui-divider"></div>
+            <div class="iui-header-divider"></div>
 
             <!-- Project Button -->
             <nav aria-label="breadcrumbs">
               <span class="iui-button-split-menu iui-header-split-button">
```

Comment:
Now it's a weird situation where we have a mix of our readymade button components and these new header button variants. Will need to work on simplifying this split button probably